libs/shared/design-tokens/generated/patterns/button.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * THIS FILE IS GENERATED BASED ON THE DESIGN TOKENS DEFINED IN THE src/**.yml
+ * FILES, DO NOT CHANGE MANUALLY.
+ * TO GENERATE THESE FILES RUN `ng build shared-design-tokens`
+ */
+
+export const FLUID_BUTTON_PADDING = '0 16px';
```

Comment:
Just as a comment, would it make sense to transform these padding / spacing tokens into an object maybe like 
```
{ top: { value: 0} , right: { value: 16, unit: 'px' } ...
```

Reply:
I guess it could make sense to split them, but let's think about the use case of the design tokens in typescript here. In 99% of the cases, you are going to mirror the value of these tokens into a style attribute or something similar. If we split the values, a consumer would have to join these together manually again. 
I don't think we need to split these values up.

Also the parsing of multi-value css or other shorthands can be quite complicated.